Bottom: 5955592b21d684213b510d83627ae9ac35cd5ec7
Top:    b8ddccfd091d6490cf86bd3ed474cac9f6afc28d
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-14 23:18:56 +0100

Refresh of find-c-remove-stray-restoreund

---

diff --git a/src/find.c b/src/find.c
index 6edd29d..877deb4 100644
--- a/src/find.c
+++ b/src/find.c
@@ -3534,7 +3534,6 @@ DRCFind (int What, void *ptr1, void *ptr2, void *ptr3)
           drc = false;
           Bloat = -PCB->Shrink;
           TheFlag = SELECTEDFLAG;
-          RestoreUndoSerialNumber ();
           ListStart (What, ptr1, ptr2, ptr3);
           DoIt (true, true);
           DumpList ();
@@ -3588,7 +3587,6 @@ DRCFind (int What, void *ptr1, void *ptr2, void *ptr3)
       User = true;
       drc = false;
       Bloat = 0;
-      RestoreUndoSerialNumber ();
       TheFlag = SELECTEDFLAG;
       ListStart (What, ptr1, ptr2, ptr3);
       DoIt (true, true);
